fbw-common/src/wasm/systems/systems/src/air_conditioning/acs_controller.rs

```diff
@@ -858,8 +858,44 @@ impl<const ENGINES: usize> PackFlowController<ENGINES> {
         }
     }
 
-    fn fcv_status_determination(&self, pneumatic: &impl PackFlowValveState) -> bool {
-        (pneumatic.pack_flow_valve_is_open(self.id + 1)) != self.fcv_open_allowed
+    fn fcv_fault_determination(&self) -> bool {
+        self.fcv_disagree_status()
+            || (self.fcv_open_allowed && self.inlet_pressure_below_min.output())
+    }
+
+    fn update_pressure_condition(
+        &mut self,
+        context: &UpdateContext,
+        pneumatic: &impl PackFlowValveState,
+    ) {
+        self.inlet_pressure_below_min.update(
+            context,
+            pneumatic
+                .pack_flow_valve_inlet_pressure(self.id + 1)
+                .map_or(false, |p| {
```

Comment:
Am I correct that this means that when the inlet pressure sensor fails/is unpowered the inlet pressure is assumed to be above min?

Reply:
Yes, this is correct and intended 👍.